coalib/processes/communication/InterruptProcess.py

```diff
@@ -0,0 +1,32 @@
+import os
+import signal
+import platform
+
+if platform.system() == "Windows":
+    from ctypes import windll
+    from time import sleep
+    CTRL_C_EVENT = 0x0
+
+
+def interrupt_process(pid):
+    """
+    Interrupt a process by its pid.
+
+    Note for Windows users:
+    Be sure that the pid you pass shares the same process group as your program
+    that calls this function. If they don't share the same process group, this
+    call blocks for 10 seconds to be sure that this is the case.
```

Comment:
If they don't share the same process group it blocks *at least* 10 secs because the interrupt will never come.
But maybe I should add a note that this call blocks until the KeyboardInterrupt was delivered...